operator/src/main/scala/io/cloudstate/operator/OperatorConstants.scala

```diff
@@ -69,6 +69,10 @@ object OperatorConstants {
   final val DefaultUserPort = 8080
   final val UserPortEnvVar = "PORT"
 
+  final val MetricPortName = "metric-port"
+  final val DefaultMetricPort = 9090
+  final val MetricPortEnvVar = "METRICS"
```

Comment:
If you were taking a poll, I'd vote for these names to use "metrics" plural...  e.g. `MetricsPortName`.

Reply:
👍 consistently using `metrics` sounds good